src/integrations/ga.js

```diff
@@ -1,9 +1,15 @@
+import { generate } from '../guids.js';
+
 let pollAttempt = 0;
 const maxPollAttempts = 60;
 
 function gaIntegration() {
     if (pollAttempt > maxPollAttempts) {
-        console.log('Evolv: unable to set uid');
+        console.log('Evolv: GA clientId not found, using generated UID');
+        var id = generate();
```

Comment:
Nitpick: prefer `const` whenever possible over `let` and definitely over `var` (IE11 does support `const`)
```suggestion
        const id = generate();
```

Reply:
Updated to `const`